tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1145,6 +1145,54 @@ TEST(onnx, importSliceNoAxes) {
                   {2, 1, 2, 2} /* output */);
 }
 
+static void importCast(std::string fileName, const char *inputName,
+                       const llvm::ArrayRef<size_t> inputShape,
+                       ElemKind outputKind) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetFilename =
+      std::string(GLOW_DATA_PATH "tests/models/onnxModels/") + fileName;
+  PlaceholderBindings bindings;
+  Placeholder *graphOutputVar;
+  {
+    Tensor data;
+    getNCHWData(&data, inputShape[0], inputShape[1], inputShape[2],
+                inputShape[3]);
+    ONNXModelLoader onnxLD(NetFilename, {inputName}, {&data.getType()}, *F);
+    graphOutputVar = EXIT_ON_ERR(onnxLD.getSingleOutput());
+    bindings.allocate(mod.getPlaceholders());
+    updateInputPlaceholdersByName(bindings, &mod, {inputName}, {&data});
+  }
+
+  // ONNX importer loads a Cast operator and adds to the IR:
+  // - a ConvertTo node
+
+  // Check the graph structure.
+  auto *saveNode = getSaveNodeFromDest(graphOutputVar);
+  auto *node = saveNode->getInput().getNode();
+  auto *castNode = llvm::dyn_cast<ConvertToNode>(node);
+  EXPECT_NE(nullptr, castNode);
```

Comment:
nit: ASSERT_TRUE to abort further execution as it's pointless

Reply:
Done. Used `ASSERT_NE`.